ds/outputs/data/processed/_articles/deep_learning_count.json

```diff
@@ -0,0 +1,41 @@
+{
+"api_doc_url": "DAPS",
+"api_type": "MySQL",
+"data_date": "2021/07/01",
+"description": "Number of preprints related to an AI category",
+"endpoint_url": "DAPS",
+"framework_group": "publications",
+"is_experimental": true,
+"order": [
+"year",
+"region_type",
+"region_year_spec",
+"region_id",
+"region_level",
+"value.id"
+],
+"region_types": [
+"NUTS"
+],
+"schema": {
+"region_id": "string",
+"region_level": "int",
+"region_type": "string",
+"region_year_spec": "int",
+"value": {
+"data_type": "int",
+"format": ".1f",
+"id": "deep_learning_count",
+"label": "Number participations in Deep Learning preprints"
+},
+"year": {
+"data_type": "int",
+"label": "Year"
+}
+},
+"source_name": "Nesta",
+"source_url": "https://github.com/nestauk/eurito_indicators",
+"subtitle": "Number participations in Deep Learning preprints by institutions in the location (papers identified through a semantic analysis of abstract topics",
+"title": "Number of participations in Deep Learning preprints",
+"date": "2021-12-16"
```

Comment:
(5) `date` is not a field in the schema.
If this was supposed to be `data_date`, then we have a duplicate info here